packages/client3/components/pool/pool-information/pool-information.tsx

```diff
@@ -127,6 +125,10 @@ export function PoolInformation({
       });
     } else {
       const now = (await provider.getBlock("latest")).timestamp;
+      /**
+       * Note : Sometime the timestamp returned by the provider(metamask) is of old blocks and would cause 1hr deadline to fail.
+       * Increasing the deadline seems to fix it but not the ideal approach.
```

Comment:
Not that it matters but can you include a link to this? I know we spoke about it but just for reference.